robottelo/constants.py

```diff
@@ -255,7 +255,8 @@
 #: Name (not label!) of the default location.
 DEFAULT_LOC = "Default Location"
 DEFAULT_CV = "Default Organization View"
-DEFAULT_SUBSCRIPTION_NAME = 'Employee SKU'
+DEFAULT_SUBSCRIPTION_NAME = (
+    'Red Hat Enterprise Linux Server Entry Level, Self-support')
```

Comment:
Just fyi, this is to support recent changes which changed our subscriptions and therefore broke our automation.